web-admin/src/routes/+layout.ts

```diff
@@ -23,7 +23,8 @@ import { fixLocalhostRuntimePort } from "@rilldata/web-common/runtime-client/fix
 import { runtime } from "@rilldata/web-common/runtime-client/runtime-store";
 import { error, redirect, type Page } from "@sveltejs/kit";
 
-export const load = async ({ params, url, route }) => {
+export const load = async ({ params, url, route, depends }) => {
+  depends("init");
```

Comment:
Can we call this "root", just so we don't confuse it with Rill Developer's "init" loader (which is called "init" because it has "initialization" logic that redirects users to the Welcome page)